addons/common/functions/fnc_getVehicleIcon.sqf

```diff
@@ -35,7 +35,11 @@ if (isNil "_cachedValue") then {
         if (_vehicleValue != "" && {((toLowerANSI _vehicleValue) find ".paa") > -1}) then {
             _cachedValue = _vehicleValue;
         } else {
-            _cachedValue = DEFAULT_TEXTURE;
+            if (fileExists (_vehicleValue + ".paa")) then {
```

Comment:
I would handle `""` directly, because it's easy to filter out, but otherwise good imo.
```suggestion
            if (_vehicleValue != "" && {fileExists (_vehicleValue + ".paa")}) then {
```

Reply:
It's more overhead to check for empty strings than it is to just do it.
![image](https://github.com/user-attachments/assets/6bd5f24b-dc7b-4173-80c5-0c39f80b8d7a)
![image](https://github.com/user-attachments/assets/40aba9e8-ca5b-4aff-9b5f-24d04ffdc53b)

Reply:
Except you aren't checking if it's `""`, you would be checking `".paa"`, which isn't the same.